packages/backend-app-api/src/logging/VoidLogger.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  LoggerService,
+  RootLoggerService,
+} from '@backstage/backend-plugin-api';
+import { JsonObject } from '@backstage/types';
+
+/**
+ * A empty {@link @backstage/backend-plugin-api#LoggerService} implementation.
+ *
+ * @public
+ */
+export class VoidLogger implements RootLoggerService {
```

Comment:
>[!Note] Created this to be used by the Postgress search engine

Reply:
Hmm, I get why you added it, but I don't think we ever want plugins to use this in the long run. If optional logging is desired then I think it's better to simply `logger?.info(...)`. I think moving this to where it is used in the search plugin should be alright for now, until we can remove it from there too?

Reply:
Update to use `logger?.` 🙌🏻